config/runtimes/kserve-mlserver.yaml

```diff
@@ -6,13 +6,18 @@ spec:
   supportedModelFormats:
     - name: sklearn
       version: "0"
+      autoSelect: true
     - name: xgboost
       version: "1"
+      autoSelect: true
     - name: lightgbm
       version: "3"
+      autoSelect: true
     - name: mlflow
       version: "1"
       autoSelect: true
+  protocolVersions:
+    - v2
```

Comment:
- v2
- grpc-v2

Reply:
for serverless mode it can only be either rest or grpc, but for raw deployment you can actually do both rest and grpc in the same serving runtime.